packages/components/text-field/src/text-field.ts

```diff
@@ -140,13 +140,13 @@ export class TextField<T extends { toString(): string } = string>
    */
   @property() type: 'email' | 'number' | 'tel' | 'text' | 'url' | 'password' = 'text';
```

Comment:
Can you please also remove 'number' from here since we already have a number-field? :)